packages/app/src/app/components/CreateSandbox/Import/Import.tsx

```diff
@@ -162,14 +170,11 @@ export const Import: React.FC<ImportProps> = ({ onRepoSelect }) => {
         Enter the GitHub repository URL to import
       </Text>
       {hasMaxPublicRepositories ? <MaxPublicRepos /> : null}
+      {!ghAuth.allowsPublicRepos ? <UnauthorizedGitHub /> : null}
       <Element
-        {...(hasMaxPublicRepositories
+        {...(disableImport
           ? {
               as: 'div',
-              css: {
```

Comment:
Now that the `css` is removed from this element and we disable the `input`, do we still need to use `as: 'div'`? I think a form is fine. What do you think?

Reply:
[9347827](https://github.com/codesandbox/codesandbox-client/pull/7385/commits/9347827d661b3c1b5f8798362aa2cee6c9a68c04)